### PR TITLE
chore: enforce conventional commits

### PR DESCRIPTION
**Pull request by mwwoda**

### CODE REVIEW

File: .github/semantic.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+titleOnly: true
```
Comment (lukaszsocha2) [inline .github/semantic.yml:1]: no new line

Comment (lukaszsocha2) [reply .github/semantic.yml:1]: So if there will be anything in commit message body it will fail ?

Comment (mwwoda) [reply .github/semantic.yml]: Added

**Review by lukaszsocha2 — APPROVED** [review]
Looks good

**Pull request merged**